db/schema.rb

```diff
@@ -171,6 +171,7 @@
     t.integer  "project_id",    null: false, default: 0
     t.datetime "announced",     null: false
     t.string   "description",   null: false, default: ''
+    t.string   "short_desc",    null: true, default: ''
```

Comment:
```suggestion
    t.string   "short_desc",    null: false, default: ''
```

In general I prefer `null: false` so that the database can predict the size of a row easier.